internal/help/help.go

```diff
@@ -93,7 +93,9 @@ func WriteToolVersionHelp(conf config.Config, toolName, toolVersion string, writ
 func writePluginHelp(conf config.Config, toolName, toolVersion string, writer io.Writer, errWriter io.Writer) error {
 	plugin := plugins.New(conf, toolName)
 	env := map[string]string{
-		"ASDF_INSTALL_PATH": plugin.Dir,
+		"ASDF_INSTALL_PATH":    plugin.Dir,
+		"ASDF_INSTALL_VERSION": "",
+		"ASDF_INSTALL_TYPE":    "",
```

Comment:
Why do the changes above not unset these two variables?

Reply:
D'oh. I thought I added these to a test file. Let me see if we can do without.

Reply:
Could you re-test with your setup to confirm whether any of the help tests fail?